server/@types/educationAndWorkPlanApi/index.d.ts

```diff
@@ -2313,7 +2365,7 @@ export interface components {
        */
       scheduleStatus:
         | 'SCHEDULED'
-        | 'COMPLETE'
+        | 'COMPLETED'
```

Comment:
The change of Induction status from `COMPLETE` to `COMPLETED`, plus a couple of other new ones